### ISSUE TITLE
Model behaving different on different machine

### ISSUE THREAD

**Issue by gifflarn** [guillaumegenthial/sequence_tagging#44]
Hi!

I used the program and ran all the necessary steps on a machine with access to a beefy GPU. With test results around acc: 97% and f1 score: 91% I found it to be great. However, when trying to further implement other stuff, according to my needs, on a computer without the GPU required I figured I'd just transfer the model.weights folder along with chars.txt, tags.txt and words.txt from the GPU computer to this one. What I first noticed when i ran evaluate.py is that I got a error saying that lhs shape did not match rhs shape, which I 'fixed' by running build_data.py (which might or might not be part of the issue).

When running evaluate.py I'm getting the same accuracy and f1 scores as I would running the test.txt as both train and test set.

What's bothering me is that the save / reload works perfectly on the GPU computer but not at all on my computer.

GPU comp runs tensorflow-gpu and python version 3.6.1 while my computer runs 'regular' tensorflow with python version 3.6.5. Could the difference in versions be the source of the problem?

Regards, Lukas

**Comment by grafael** [comment]
Check **tensorflow** version on you machine. I mean, your **tensorflow-gpu** and **tensorflow** (without GPU) should br the same.

**Comment by grafael** [comment]
Check **tensorflow** version on your machine. I mean, your **tensorflow-gpu** and **tensorflow** (without GPU) should be the same.

**Comment by gifflarn** [comment]
The GPU machine runs tensorflow-gpu 1.5.0 and my machine runs tensorflow 1.5.0.

**Comment by gifflarn** [comment]
With tensorflow upgraded to 1.7 on both devices, this no longer is an issue
